### PR TITLE
Update main resources and equipment page to update info on HPC

### PR DESCRIPTION
**Pull request by willfurnass**
To review/update:

- [x] pages/community/resources-and-equipment/index.md
- [x] pages/community/resources-and-equipment/languages/fortran.md
- [x] pages/community/resources-and-equipment/languages/R.md
- [x] pages/community/resources-and-equipment/languages/mathematica.md
- [x] pages/community/resources-and-equipment/languages/MATLAB.md
- [x] pages/community/resources-and-equipment/languages/maple.md
- [x] pages/community/resources-and-equipment/languages/NAG.md
- [x] pages/training/iceberg.md
- [x] pages/training/programming/index.md
- [x] pages/service/activities/activities.md
- [x] pages/service/testimonials/testimonials.md
- [x] _posts/2018-02-10-ssi_fellowships.md
- [x] _posts/2016-05-23-fun-with-strace.md
- [x] _posts/2017-06-15-tmux-intro.md
- [x] _posts/2017-05-08-spark-scala-hpc.md
- [x] _posts/2016-05-26-HPC-maple.md
- [x] _posts/2016-07-02-the-university-of-sheffield-named-an-nvidia-gpu-education-center.md
- [x] _posts/2017-11-23-sharc_vs_iceberg.md
- [x] _posts/2017-01-30-willfurnass_intro.md
- [x] _posts/2020-05-20-ssh-best-prac.md
- [x] _posts/2016-09-12-intel_R_iceberg.md

### CODE REVIEW

**Comment by willfurnass** [comment]
Made some updates to the HPC-related reference info.  It could be better structured but is now at least up to date.

Keen to get this merged before the event tomorrow.

File: pages/community/resources-and-equipment/index.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ slug: index
 type: text
 ---
 
-Here you'll find details of hardware and software resources that will be of interest to computational researchers, research software developers and research software engineers at The University of Sheffield.
+Here you'll find details of hardware and software resources that will be of interest to
+computational researchers, research software developers and research software engineers at The University of Sheffield.
```
Comment (bobturneruk) [inline pages/community/resources-and-equipment/index.md]: Data Scientists?

File: pages/community/resources-and-equipment/languages/MATLAB.md [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
-MATLAB is a high-level technical computing language and interactive
-environment for algorithm development, data visualization, data
-analysis, and numerical computing. The University of Sheffield has a
-Total Academic Headcount license for MATLAB which allows both staff and
-students to use it, both on University and personally owned machines.
+MATLAB is a high-level technical computing language and interactive environment for
+algorithm development, data visualization, data analysis, and numerical computing.
+The University of Sheffield has a Total Academic Headcount license for MATLAB which
+allows both staff and students to use it,
+both on University and personally owned machines.
```
Comment (bobturneruk) [inline pages/community/resources-and-equipment/languages/MATLAB.md:20]: Really also personal machines?

**Review by bobturneruk — APPROVED** [review]
I'm not expert enough in HPC provision to confirm the accuracy of everything here, but there's clearly very important stuff being corrected here (e.g. iceberg is no longer a thing), so I advise a merge. If there are typos, we'll catch them later!

File: pages/community/resources-and-equipment/languages/MATLAB.md (hunk 2 above)
Comment (willfurnass) [reply pages/community/resources-and-equipment/languages/MATLAB.md:20]: Yep, just checked.

File: pages/community/resources-and-equipment/index.md (hunk 1 above)
Comment (willfurnass) [reply pages/community/resources-and-equipment/index.md]: Oh go on then.

**Pull request merged**